frontend/src/openapi/models/instanceAdminStatsSchema.ts

```diff
@@ -25,8 +25,8 @@ export interface InstanceAdminStatsSchema {
      * @minimum 0
      */
     contextFields?: number;
-    /** The rounded up average number of edge instances in the last month and month before last */
-    edgeInstances?: InstanceAdminStatsSchemaEdgeInstances;
+    /** The average number of edge instances, per month, in the last 12 months, rounded to 3 decimal places */
+    edgeInstanceAveragesPerMonth?: InstanceAdminStatsSchemaEdgeInstanceAveragesPerMonth;
```

Comment:
Damn that's a mouthful of a name. What about EdgeInstanceUsage or something a bit less paragraph-as-a-name?